Sources/TuistSupport/Constants.swift

```diff
@@ -30,7 +30,7 @@ public enum Constants {
     /// The cache version.
     /// This should change only when it changes the logic to map a `TuistGraph.Target` to a cached build artifact.
     /// Changing this results in changing the target hash and hence forcing a rebuild of its artifact.
-    public static let cacheVersion = "1.0.0"
+    public static let cacheVersion = "2.0.0"
```

Comment:
Why should it change?

Reply:
I'm glad you mentioned this. I don't think it needs to. *I* needed to do it locally, because I _changed_ the way I had implemented the public API. However, thinking on it further, any consumers of the existing API won't hit the same issue because it's changing for them in a non-breaking way. I'll revert!